src/dynamodb/dbHelper.ts

```diff
@@ -42,3 +43,11 @@ export const parseDynamoDBAttribute = <T>(
   if (!attributes || attributes === {}) return undefined;
   return attributes as T;
 };
+
+export const getHandByConnectionId = (hands: UserHand[], connectionId: string): string[] => {
+  const userHand = hands.find((hand) => {
+    return hand.connectionId === connectionId;
+  }) as UserHand;
```

Comment:
I'm surprised eslint doesn't make you do this:
```suggestion
  const userHand = hands.find((hand) => hand.connectionId === connectionId) as UserHand;
```

Reply:
I did that on purpose, cuz I think the line is a bit too long, but I can change to that

Reply:
Not important, was just curious